maxim/logger/livekit/agent_activity.py

```
@@ -0,0 +1,186 @@
+    # lets check if the current turn has any assistant response
+    # if not - in that case user interrupted the turn - so we can go on in the current turn    
+    
+    session_info = get_session_store().get_session_by_agent_session_id(
+        id(self._session)
+    )
+    if session_info is None:
+        scribe().error("[MaximSDK] session info is none at realtime session emit")
+        return
+    turn = session_info.get("current_turn", None)
+    if turn is not None:
+        if turn["is_interrupted"]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)